examples/jsm/loaders/3MFLoader.js

```diff
@@ -21,7 +21,7 @@ import {
 } from 'three';
 import * as fflate from '../libs/fflate.module.js';
 
-const COLOR_SPACE_3MF = SRGBColorSpace;
+const colorspace_3MF = SRGBColorSpace;
```

Comment:
Can we keep these constants in upper case, please?

Reply:
ugh... global replace error.